config/rubocop.yml

```diff
@@ -92,3 +92,8 @@ AllCops:
 Style/BlockDelimiters:
   Exclude:
     - "spec/**/*"
+
+# Ruby version
+
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
@grk [it works like this](https://github.com/Neodelf/rubocop/commit/749b86166c3e0dee332114f7ca7685c8d138826e) - take `.ruby-version` if exists but fall back to the `rubocop.yml`. Unfortunately, it does not consider `Gemfile` (imho this would be the best - point the right version in the file where you have all dependencies defined).